tox_pipenv/plugin.py

```diff
@@ -7,80 +8,61 @@
 import contextlib
 
 
-def _init_pipenv_environ():
-    os.environ["PIPENV_ACTIVE"] = "1"
+ENV_PIPENV_INSTALL_ARGS = "TOX_PIPENV_INSTALL_ARGS"
+
+
+def _env_install_args():
+    args_str = os.environ.get(ENV_PIPENV_INSTALL_ARGS)
+    if args_str:
+        return list(shlex.split(args_str))
 
-    # Ignore host virtual env
-    os.environ["PIPENV_IGNORE_VIRTUALENVS"] = "1"
 
+def _init_pipenv_environ():
     # Answer yes on recreation of virtual env
     os.environ["PIPENV_YES"] = "1"
 
     # don't use pew
     os.environ["PIPENV_VENV_IN_PROJECT"] = "1"
 
+    # don't print the warning about using an existing venv
+    os.environ["PIPENV_VERBOSITY"] = "-1"
+
 
 def _clone_pipfile(venv):
     if hasattr(venv, 'session'):
         root_pipfile_path = venv.session.config.toxinidir.join("Pipfile")
     else:
         root_pipfile_path = venv.envconfig.config.toxinidir.join("Pipfile")
```

Comment:
Yes good point. The plug-in uses that env var internally since it copies the Pipfile from its original location into the venv... but there's no reason not to respect it if the user provides it.